debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -209,6 +217,14 @@ public String getAlias() {
                     " '${routedByValue}' is available which is the value of The column configured" +
                     " via 'route.by.field'");
 
+    static final Field ROUTE_TOMBSTONE_ON_EMPTY_PAYLOAD = Field.create("route.tombstone.on.empty.payload")
```

Comment:
Ok, I'm a bit confused, as that seems not to be what we discussed. But using an empty/null payload as the trigger to produce a tombstone is fine with me, too. I'm only wondering then:

* Why do we need FIELD_EVENT_DELETED?
* Why do we need allowDeleteOperations?

It seems ROUTE_TOMBSTONE_ON_EMPTY_PAYLOAD is good enough, no? If true and when receiving an empty event payload, a tombstone will be emitted, and that's it. Or am I missing anything?

Reply:
If we go for the empty payload indeed it's the only needed option!

I also got a bit confused @gunnarmorling , would you explain again how would your solution look like?

Reply:
I think this was misunderstanding on my part. Looking back it looks like we do need only the single config option and additional fields attribute can be used to convey they is_deleted property on a message.

Reply:
Exactly, @brbrown25! I.e. FIELD_EVENT_DELETED and allowDeleteOperations can be removed, and then this should be good to go.